jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -64,42 +61,188 @@ The link:https://circleci.com/developer/orbs/orb/circleci/evals[official Evals o
 If you run evaluations using a different tool, let us know at mailto:ai-feedback@circleci.com[]. You can also contribute directly to the official orb by opening a PR on the link:https://github.com/CircleCI-Public/ai-evals-orb[public repository].
```

Comment:
Is this section still accurate? Can people just integrate with any third party tool with v2 of the orb?

Reply:
Good catch - no longer accurate, but more so because the repo is now private and not public.